unimod_mapper/unimod_mapper.py

```diff
@@ -62,6 +62,16 @@ def __init__(self, refresh_xml=False, xml_file_list=None):
 
         self.unimod_xml_names = ["unimod.xml", "usermods.xml"]
```

Comment:
Could be a class attribute.
That would mean, you would not need to create a new instance of the mapper just to access the (static) attribute [here](https://github.com/fu/ursgal2/pull/119)

Reply:
Forget it, just figured out this would create unwanted side effects